src/frontend/ExtensionHandler/index.tsx

```diff
@@ -5,6 +5,7 @@ import { useNavigate } from 'react-router-dom'
 
 declare global {
   interface Window {
+    // @ts-expect-error: conflict with wagmi ethereum types
```

Comment:
If we don't do this, the global type definition of wagmi conflicts with this.

Reply:
Can't we extend the original type on a new type modifying the needed props?

Reply:
it's a bit complicated and not worth it IMO, wagmi is expecting some TS const assertions, our definitions are similar but without the assertions.

Reply:
Actually, I was able to remove the expect error :D